packages/api/src/controllers/webhook.test.js

```diff
@@ -190,6 +190,18 @@ describe("controllers/webhook", () => {
       expect(updated.name).toEqual("modified_name");
     });
 
+    it("update the status of a webhook", async () => {
+      const { id } = generatedWebhook;
+      const payload = {
+        status: {
+          lastTriggeredAt: Date.now(),
+          lastFailure: { timestamp: Date.now() },
+        },
+      };
```

Comment:
You should try sending a response here as well to catch 1 issue that you likely haven't noticed 👀 

The `webhook-response` object has a required `eventId` because that's how it works for all webhook calls right now. I think we should make that field optional instead, just so the client doesn't have to generate some random ID just to fill that field.